streams/src/main/java/org/apache/kafka/streams/processor/api/ProcessorContext.java

```diff
@@ -106,7 +105,7 @@ void register(final StateStore store,
 
     /**
      * Schedules a periodic operation for processors. A processor may call this method during
-     * {@link Processor#init(org.apache.kafka.streams.processor.ProcessorContext) initialization} or
+     * {@link Processor#init(ProcessorContext) initialization} or
```

Comment:
Removing the package name to point to `org.apache.kafka.streams.processor.api.ProcessorContext` (note the `api` sub-package in the path)

Reply:
Good catch.